README.md

```diff
@@ -105,9 +108,14 @@ A programming idiom is a means of expressing a recurring construct in one or mor
 * the construction process must allow different representations for the object that's constructed
 
 **Real world examples:**
-* [java.lang.StringBuilder](http://docs.oracle.com/javase/8/docs/api/java/lang/StringBuilder.html)
+* [java.lang.StringBuilder#append() (unsynchronized)] (http://docs.oracle.com/javase/8/docs/api/java/lang/StringBuilder.html)
```

Comment:
Please link only the method i.e. 

[java.lang.StringBuilder#append() (unsynchronized)] (http://docs.oracle.com/javase/8/docs/api/java/lang/StringBuilder.html)

change to:

[java.lang.StringBuilder#append()](http://docs.oracle.com/javase/8/docs/api/java/lang/StringBuilder.html) (unsynchronized)